k8s/aws-k8s-steps.adoc

```diff
@@ -0,0 +1,62 @@
+== Kubernetes Deployment in AWS EKS (Elastic Kubernetes Service)
+
+=== Pre-requisites
```

Comment:
Maybe we can also add here that the AWS CLI (and Azure CLI in the other file) needs to be installed and add a link to the corresponding installation guide?